datafusion/physical-plan/src/aggregates/row_hash.rs

```diff
@@ -529,7 +529,12 @@ impl GroupedHashAggregateStream {
             })
             .collect();
 
-        let name = format!("GroupedHashAggregateStream[{partition}]");
+        let agg_fn_names = aggregate_exprs
+            .iter()
+            .map(|expr| expr.name().to_string())
```

Comment:
nit: have you tried out `human_display()`? It shows `count(1)` instead of `count(Int64(1))` in this case, wondering if it's better overall for error messages or not.